Dockerfile

```diff
@@ -49,6 +49,8 @@ COPY --from=build /usr/local/bin/gem /usr/local/bin/gem
 COPY --from=build /usr/local/bundle/ /usr/local/bundle/
 COPY --from=build /app/ /app/
 
+RUN bundle config set without 'development test assets'
```

Comment:
Why are we adding this in prod image?

Reply:
to not load `assets` group, it is loaded in build phase to be able to run `assets:precompile`

Reply:
I mean to ask will this config ever be in use in the prod image? we don't run bundle install in prod.

Reply:
ahh, right, I can try without this